src/utils/request-queue.js

```diff
@@ -43,87 +47,93 @@ class RequestQueue {
 
   // Read from queue and send requests to server
   send() {
-    // Defer sending of events to give beforeunload time to register (avoids race condition)
-    setTimeout(() => {
-      const fetch = (this.options && this.options.fetch) || fetchPonyfill({ Promise }).fetch;
-      const queue = RequestQueue.get();
-
-      if (
-        this.humanity.isHuman()
-        && !this.requestPending
-        && !this.pageUnloading
-        && queue.length
-      ) {
-        let request;
-        let nextInQueue = queue.shift();
-
-        RequestQueue.set(queue);
-
-        // Backwards compatibility with versions <= 2.0.0, can be removed in future
-        // - Request queue entries used to be strings with 'GET' method assumed
-        if (typeof nextInQueue === 'string') {
-          nextInQueue = {
-            url: nextInQueue,
-            method: 'GET',
-          };
-        }
-
-        if (nextInQueue.method === 'GET') {
-          request = fetch(nextInQueue.url);
-        }
-
-        if (nextInQueue.method === 'POST') {
-          request = fetch(nextInQueue.url, {
-            method: nextInQueue.method,
-            body: JSON.stringify(nextInQueue.body),
-            mode: 'cors',
-            headers: { 'Content-Type': 'text/plain' },
-          });
-        }
-
-        if (request) {
-          this.requestPending = true;
-          const instance = this;
+    if (this.sendTrackingEvents) {
```

Comment:
There may be legal implications here that are problematic... we should probably revert this change